packages/mantine/src/components/table/layouts/RowLayout.tsx

```diff
@@ -158,6 +158,7 @@ const RowLayoutBody = <T,>({
                                     width: columnSizing.size ?? 'auto',
                                     minWidth: columnSizing.minSize,
                                     maxWidth: columnSizing.maxSize,
+                                    wordBreak: 'break-word',
```

Comment:
Can you use `overflowWrap: 'break-word'` instead? It's the newer version of `wordBreak: break-word`

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
works flawlesly